packages/core/src/app/checkout/CheckoutStepHeader.tsx

```diff
@@ -29,10 +29,10 @@ const CheckoutStepHeader: FunctionComponent<CheckoutStepHeaderProps> = ({
     type,
 }) => {
     return (
-        <a
```

Comment:
For accessibility, wouldn't screen readers prefer having the `a` tag as opposed to a clickable `div` tag

Reply:
The current issue is that a screen reader might mislead a shopper by announcing our step header as a clickable link.
Regarding your concern, all editable (collapsed) steps have delicate `edit` buttons, it should be good to use them.

Reply:
Do we need to change the html element here?

Reply:
Think we have to get rid of `a` so that a screen reader won't annouce each step header as a link.

I'll add an expanded or collapsed state aria attribute in the next PR.